indico/modules/events/management/settings.py

```diff
@@ -22,3 +23,7 @@
     'privacy_policy_urls': [],
     'privacy_policy': '',
 })
+
+misc_settings = SettingsProxy('misc', {
```

Comment:
How about `events_settings` and `events`? I think for "generic settings related to events" that fits much better than `misc`...

Reply:
Renamed with your suggestion.